x-pack/plugins/lens/public/datasources/form_based/operations/definitions/column_types.ts

```diff
@@ -35,6 +35,7 @@ export interface ValueFormatConfig {
 // Formatting can optionally be added to any column
 export interface FormattedIndexPatternColumn extends BaseIndexPatternColumn {
   params?: {
+    window?: number;
     format?: ValueFormatConfig;
   };
 }
```

Comment:
When defining the moving average definition like here...

https://github.com/elastic/kibana/blob/a20a96c6fb862a05f81c64a96e200432a4677c4a/x-pack/plugins/lens/public/datasources/form_based/operations/definitions/formula/formula.test.tsx#L58-L74

The type of `params: { window: 5 }` is pointing to this type here thus why adding the `window` prop fixes the error.

But I think `window` type should be pulling from `MovingAverageIndexPatternColumn` instead which is here...

https://github.com/elastic/kibana/blob/a20a96c6fb862a05f81c64a96e200432a4677c4a/x-pack/plugins/lens/public/datasources/form_based/operations/definitions/calculations/moving_average.tsx#L45-L51